submissions/BlueLamantine/html-css-popup/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="shortcut icon" href="img/favicon.png" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <nav class="topnav">
+      <ul class="topnav-list">
+        <li class="list-item">
+          <a href="#" class="nav-item">Mail</a>
+        </li>
+        <li class="list-item">
+          <a href="#" class="nav-item">Images</a>
+        </li>
+        <li class="list-item">
+          <input id="menu-toggle" type="checkbox" />
```

Comment:
Watch [the video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) about how to hide checkbox in the right way

Reply:
Thanks a lot for this useful information! This method has been implemented in my work.